dev/com.ibm.ws.security.token.ltpa_fat/fat/src/com/ibm/ws/security/token/ltpa/fat/LTPAInitializationVectorTests.java

```diff
@@ -0,0 +1,1476 @@
+        renameFileIfExists(DIFFERENT_PW_VALIDATION_KEY_PATH, DEFAULT_KEY_PATH, true, server1);
+        // Wait for the LTPA configuration to be ready after the change
+        assertNotNull("Expected LTPA configuration ready message not found in the log.",
+                      server1.waitForStringInLog("CWWKS4107A", 5000));
+
+        configureServer("true", "10", true, server2);
+        renameFileIfExists(VALIDATION_KEY2_PATH, DEFAULT_KEY_PATH, true, server2);
+        // Wait for the LTPA configuration to be ready after the change
+        assertNotNull("Expected LTPA configuration ready message not found in the log.",
+                      server2.waitForStringInLog("CWWKS4107A", 5000));
+
+        // Change the default keysPassword to that of the added ltpa keys file (encrypted "Liberty")
+        ServerConfiguration serverConfig = server1.getServerConfiguration();
+        LTPA ltpa = serverConfig.getLTPA();
+        setLTPAKeyPasswordElement(ltpa, "{xor}EzY9Oi0rJg==");
+        // Wait for the LTPA configuration to be ready after the change
+        assertNotNull("Expected LTPA configuration ready message not found in the log.",
+                      server1.waitForStringInLog("CWWKS4107A", 5000));
+        updateConfigDynamically(server1, serverConfig);
+
+        // Initial login to simple servlet for form login1
+        String response1 = server1FlClient1.accessProtectedServletWithAuthorizedCredentials(FormLoginClient.PROTECTED_SIMPLE, validUser, validPassword);
+
+        // Get the SSO cookie from the login
+        String server1Cookie = server1FlClient1.getCookieFromLastLogin();
+        assertNotNull("Expected SSO Cookie 1 is missing.", server1Cookie);
+
+        // Copy the ltpa.keys file to server #2 and rename it as configuredValidation1.key (server #2 contains the ltpa.keys file from server #1 as a validation key)
+        copyFileToServerResourcesSecurityDir(ALT_VALIDATION_KEY9_PATH, server2);
+        renameFileIfExists(DIFFERENT_PW_VALIDATION_KEY_PATH, CONFIGURED_VALIDATION_KEY1_PATH, true, server2);
+      
+
+        // // Change the default password of the validation key to another password (encrypted "mypassword")
+        ServerConfiguration server2Config = server2.getServerConfiguration();
+        LTPA ltpa2 = server2Config.getLTPA();
+        setLTPAvalidationKeyPasswordElement(ltpa2, "{xor}MiYvPiwsKDAtOw==");
+        setLTPAKeyPasswordElement(ltpa2, "{xor}MiYvPiwsKDAtOw==");
+        
+
+        // Attempt to login to the simple servlet on server #2 and assert that the login is a failure as expected
+        //IV is different for both servers.
+        assertTrue("An invalid cookie should result in authorization challenge",
+                   server2FlClient1.accessProtectedServletWithInvalidCookie(FormLoginClient.PROTECTED_SIMPLE, server1Cookie));
+    }
+
+    /**
+     * Negative Test Case:
+     * 1. Server #1 and Server #2 contain different primary LTPA keys with different LTPA keys passwords
```

Comment:
don't they have different keys with the same passwords? the server.xml has the same password for both.